src/main/java/com/faforever/api/data/listeners/VotingSubjectEnricher.java

```diff
@@ -120,4 +113,27 @@ void calculateWinners(VotingQuestion votingQuestion) {
 
     return first.map(votingChoiceListEntry -> Collections.singletonList(votingChoiceListEntry.getKey())).orElse(Collections.emptyList());
   }
+
+  private void moveOnToTheNextAnswer(Map<VotingChoice, List<VotingAnswer>> votersByChoice, VotingAnswer votingAnswer) {
+    int newAlternativeOrdinal = votingAnswer.getAlternativeOrdinal() + 1;
+    votingAnswer.getVote().getVotingAnswers().stream()
+      .filter(votingAnswerFilter -> votingAnswerFilter.getVotingChoice() != null
+        && votingAnswerFilter.getVotingChoice().getVotingQuestion().equals(votingAnswer.getVotingChoice().getVotingQuestion())
```

Comment:
Use Objects.equals to prevent another NPE

Reply:
They should just not use the freaked out "alternative voting"